### ISSUE TITLE
Add configuration to make window stay on top of other windows

### ISSUE THREAD

**Issue by davidhemphill** [Stengo/DeskPad#21]
Cool app!

I would love to have the ability to keep the extended display window on top of all other windows. This would allow for making a video fullscreen on the extended display and mimicking Picture-in-Picture mode, when it's not available.

**Comment by lhotari** [comment]
Yes, this would be a useful feature for many usecase. This seems to be the only feature that I'm missing from DeskPad.
